app/services/prisoner_location.rb

```diff
@@ -36,6 +40,7 @@ def located_at_given_prison
   end
 
   def load_prisoner_location
+    return Nomis::Establishment.new unless offender.valid?
```

Comment:
What do you think of having `NullEstablishment` similar to the `NullOffender`?

Reply:
The naming of the `*prisoner_location` methods on this class (called `PrisonerLocation`) had me confused for a while as I thought they were returning instances of `PrisonerLocation`.

I think it would be clearer to rename them to something different like `establishment, load_establishment,...`